.github/workflows/codeql.yml

```diff
@@ -50,8 +47,8 @@ jobs:
           config-file: .github/codeql-config.yml
 
       - name: Autobuild
-        uses: github/codeql-action/autobuild@v2
         if: ${{ matrix.language == 'python' }}
+        uses: github/codeql-action/autobuild@v2
```

Comment:
This step should also not be necessary as it is a no-op for non-compiled languages.